src/native/fs/fs_napi.cpp

```diff
@@ -586,6 +624,9 @@ struct FSWorker : public Napi::AsyncWorker
             if (fs_context.Get("backend").ToBoolean()) {
                 _backend = fs_context.Get("backend").ToString();
             }
+            if (fs_context.Has("supplemental_groups")) {
+                _supplemental_groups = convert_napi_number_array_to_number_vector<gid_t>(fs_context.Get("supplemental_groups").As<Napi::Array>());
```

Comment:
I just want to make sure - in case somehow you gey empty array (`[]`) what will happen?

Reply:
it will be fine, it will just create an empty vector